website/docs/r/private_dns_zone_virtual_network_link.html.markdown

```diff
@@ -1,4 +1,5 @@
 ---
+subcategory: "Private DNS"
```

Comment:
🤔 about this would it make more sense to have these as nested sub-categories? e.g. `DNS -> Public/Private`?

Reply:
Interesting! Not sure what you mean by nested, there's only 1 level of subcategories "DNS" or "Private DNS" and then a "Resources" folder underneath that. I'll DM you a link with a current example.

Reply:
I think ideally we would want to be able to nest further:
`DNS/Public/resources`
`DNS/Private/resources`
`Recovery Services/Backup/resources`
`Recovery Services/Replication/resources`
ect

Reply:
That won't be possible in the near term, but it's something we can add to the backlog to prioritize!